frontend/taipy-gui/src/components/Taipy/Field.tsx

```diff
@@ -39,6 +40,29 @@ const Field = (props: TaipyFieldProps) => {
     const formatConfig = useFormatConfig();
 
     const className = useClassNames(props.libClassName, props.dynamicClassName, props.className);
+    const markdownComponent: Components = {
```

Comment:
interesting

Reply:
I reread the description and I thought this approach fixed the issue mentioned.  Adjusting the HTML structure is not a good idea here. @FredLL-Avaiga

Reply:
I think you're right, we don't want to mess with the HTML rendered by markdown

Reply:
I see. Is it better to close this PR and leave the issue open/blocked for now as there is no reasonable solution?